storage-iops-latency-throughput-demo/azuredeploy.json

```diff
@@ -0,0 +1,306 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountType": {
+      "type": "array",
+      "defaultValue": [ "Premium_LRS", "Standard_LRS" ],
+      "metadata": {
+        "description": "This is an array of strings which describe the type of each storage account." 
+      }
+    },
+    "storageAccountNamePrefix": {
```

Comment:
Can we move this to a var and just generate a name?  seems like that still works with your sample